python/spear/env.py

```diff
@@ -77,13 +87,13 @@ def close(self):
 
     def begin_tick(self):
         self._instance.engine_service.begin_tick()
-        self._instance.game_world_service.set_game_paused(False)
+        self._instance.game_world_service.call_function(uobject=self.gameplay_statics_default_object, ufunction=self.set_game_paused_func, args={"bPaused": False})
```

Comment:
The current name of `self.gameplay_statics_default_object` makes it seem like it's part of this class' public interface, but I don't think it should be. We should change the name as follows.

```
self._gameplay_statics_default_object
```

Reply:
Ah, my bad.  This was an oversight.